stacker/blueprints/base.py

```diff
@@ -1,11 +1,13 @@
 import copy
 import hashlib
 import logging
+from string import Template as StringTemplate
```

Comment:
I would just use `import string` here, and then use Template as `string.Template` later